### PR TITLE
Refactor app to clean up code and add additional functionality for ad…

### PR DESCRIPTION
**Pull request by marty331**
…ding contacts and opportunities

### CODE REVIEW

File: data/contacts.go [hunk 1]
```diff
@@ -21,69 +28,164 @@ type Contact struct {
...
+	PMT_Subscription_ID__c string //data['subscription_id']
+	PMT_Transaction_ID__c  string //data['charge']
+	RecordTypeId           string //settings.SUBHUB_OPP_RECORD_TYPE
+	Service_Plan__c        string //nickname
+	StageName              string //Closed Won
+}
+
+func QueryOppourtunityByFxA(user Contact) Opportunity {
+	client := CreteClient()
+	// user := queryContactByEmail(email)
```
Comment (bowlofstew) [inline data/contacts.go]: Do we want to nuke the commented out code?

File: handlers/contacts.go [hunk 2]
```diff
@@ -2,78 +2,186 @@ package handlers
...
 	"io/ioutil"
-	"log"
 	"net/http"
 )
 
+var log = logging.MustGetLogger("plowforce")
+
 type Contacts struct {
-	l *log.Logger
+	// l *logging.Logger
```
Comment (bowlofstew) [inline handlers/contacts.go]: Delete?

File: handlers/contacts.go [hunk 3]
```diff
@@ -2,78 +2,186 @@ package handlers
...
 type KeyContact struct{}
 
-func NewContacts(l *log.Logger) *Contacts {
-	return &Contacts{l}
+func NewContacts() *Contacts {
+	return &Contacts{}
+}
+
+type Opportunities struct {
+	// l *logging.Logger
```
Comment (bowlofstew) [inline handlers/contacts.go]: Delete?

File: handlers/contacts.go [hunk 4]
```diff
@@ -2,78 +2,186 @@ package handlers
...
+
+type KeyOpportunity struct{}
+
+func NewOpportunity() *Opportunities {
+	return &Opportunities{}
+}
+
+func (o *Opportunities) GetOpportunities(rw http.ResponseWriter, r *http.Request) {
+	logs.Init()
+	// Read body
```
Comment (bowlofstew) [inline handlers/contacts.go]: Needed?

File: handlers/contacts.go [hunk 5]
```diff
@@ -2,78 +2,186 @@ package handlers
...
+	b, err := ioutil.ReadAll(r.Body)
+	defer r.Body.Close()
+	if err != nil {
+		http.Error(rw, err.Error(), 500)
+		return
+	}
+	// Unmarshall
```
Comment (bowlofstew) [inline handlers/contacts.go]: Needed?

File: handlers/contacts.go (hunk 2 above)
Comment (marty331) [reply handlers/contacts.go]: Good point!  I removed the comments.

File: data/contacts.go (hunk 1 above)
Comment (marty331) [reply data/contacts.go]: Good point, removed all the commented code.

**Pull request merged**